azext_edge/edge/providers/orchestration/resources/instances.py

```diff
@@ -318,15 +319,16 @@ def enable_secretsync(
             return result_spc
 
     def show_secretsync(self, name: str, resource_group_name: str) -> Optional[dict]:
```

Comment:
now we might have more than one spc, therefore should we add spc name as a parameter for show or we show just change it to `list`?

Reply:
will switch to list now